csrc/python_frontend/segmentation.cpp

```diff
@@ -79,6 +79,167 @@ int64_t SegmentationState::setupSegmentation(
   return (int64_t)segmented_fusion_->groups().size();
 }
 
+std::unordered_map<int64_t, int64_t> SegmentationState::buildSegment(
+    FusionDefinition& other,
+    int64_t segment_id) {
+  NVF_ERROR(
+      !other.completed(),
+      "Expected an incomplete definition before translation.");
```

Comment:
I am missing what `.completed` returns here. Is it that the execution is complete?

Reply:
changed from `other` to `segment_fd`